var/spack/repos/builtin/packages/timew/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Timew(CMakePackage):
```

Comment:
From a usability standpoint I would think it would be better to have the package name closer to that of the software. 

Can you rename the parent directory (`timew`) to `time-warrior` and change the name of this package to `TimeWarrior`?

Reply:
Certainly, I can do that. (I named it this way for consistency with its sister project `taskwarrior` whose package is just called `task` but I agree -- the full name is clearer.)

In general I haven't seen it written that way with the hyphen -- would renaming the directory to `timewarrior` instead of `time-warrior` be alright?

Reply:
You can see how the first example looks in the repo at https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/adept-utils/package.py.

Reply:
Actually, after reviewing `taskd`, which is also a `taskwarrior` download *and* the URL, this your naming scheme makes more sense.